README.md

```diff
@@ -121,3 +123,33 @@ addParameters({
   },
 });
 ```
+
+### react-i18next
+
+Currently, [react-i18next doesn't support other props like `locale` except `i18n`](https://react.i18next.com/latest/i18nextprovider#i-18-nextprovider-props). If you want to use i18n storybook addon, you need to wrap `I18nProvider` using with effect hook.
```

Comment:
> you need to wrap `I18nProvider` using with effect hook

I'm not a native speaker but I think you should change this to 

> you need to wrap `I18nProvider` with useEffect hook

Reply:
I'm not a native speaker, too. thanks for the feedback (b643515)